### PR TITLE
Process Tailwind output with Vite CSS plugins

### PR DESCRIPTION
**Pull request by KrisBraun**
Transform Tailwind-generated CSS with Vite CSS plugins. `vite:css` does useful things like transforming `url()` paths and inlining images. `vite:css-post` generates bundle hashes. Before this change, the CSS bundle hash wasn't changing when the generated CSS changed.

### CODE REVIEW

**Comment by KrisBraun** [comment]
@ArnaudBarre I'm interested in your review here. In particular, do you see any cases where this won't work with other plugins correctly?

**Comment by ArnaudBarre** [comment]
Answered in Discord: this way of doing things doesn't make the CSS hash independent of the utils content. To workaround that I found that calling again the transform function of the `vite:css-post` plugin with the same id but the new content gives expected result, without the need for a fake id or placeholder. It will automatically apply all the transformation from Vite (but not from other plugins, which is fine IMO)

**Comment by KrisBraun** [comment]
@ArnaudBarre The PR has been updated to apply the `renderChunk` technique.

File: playgrounds/vite/vite.config.js
Comment (thecrypticace) [inline playgrounds/vite/vite.config.js:1]: The `package.json` in The Vite playground needs to be updated otherwise it doesn't work.

**Review by thecrypticace — CHANGES_REQUESTED** [review]

**Comment by thecrypticace** [comment]
Some small changes but otherwise looks good and appears to work quite well. 💯

File: playgrounds/vite/vite.config.js
Comment (KrisBraun) [reply playgrounds/vite/vite.config.js:1]: Unintentional change (part of testing Storybook). Backing it out.

**Review by thecrypticace — APPROVED** [review]

**Pull request merged**